src/contracts.nit

```diff
@@ -71,14 +72,20 @@ private class ContractsVisitor
 
 	var toolcontext: ToolContext
 
+	var ast_builder: ASTBuilder
+
 	# The main module
 	# It's strore to define if the contract need to be build depending on the selected policy `--no-contract` `--full-contract` or default
 	var mainmodule: MModule
 
 	# Actual visited module
 	var visited_module: AModule
 
-	var ast_builder: ASTBuilder
+	# Actual visited class
+	var visited_class: nullable AClassdef
+
+	# Actual visited class
```

Comment:
```suggestion
	# Actual visited method
```

I guess?

Reply:
This code is no longer present, I corrected it in another commit. I should have changed it in this one ...